### ISSUE TITLE
Preprocess jobs failing without specific error message

### ISSUE THREAD

**Issue by leecbaker** [Illumina/hap.py#11]
I'm trying to run a comparison of the GIAB HG001 reference VCF against some local data, and am coming up with consistent crashes in hap.py. Unfortunately, the error messages don't point towards a specific cause.

Are there any instructions for how to isolate the root cause of problems like this? Is there a way of turning on debugging messages, or really any message that could point to the cause of this error?

```2017-01-23 16:45:14,544 ERROR    Exception when running <function preprocessWrapper at 0x7f8eafc6bcf8>:
2017-01-23 16:45:14,544 ERROR    ------------------------------------------------------------
2017-01-23 16:45:14,544 ERROR    Traceback (most recent call last):
2017-01-23 16:45:14,544 ERROR      File "/opt/hap.py/lib/python27/Tools/parallel.py", line 71, in parMapper
2017-01-23 16:45:14,544 ERROR        return arg[1]['fun'](arg[0], *arg[1]['args'], **arg[1]['kwargs'])
2017-01-23 16:45:14,544 ERROR      File "/opt/hap.py/lib/python27/Haplo/partialcredit.py", line 64, in preprocessWrapper
2017-01-23 16:45:14,545 ERROR        subprocess.check_call(to_run, shell=True, stdout=tfo, stderr=tfe)
2017-01-23 16:45:14,545 ERROR      File "/opt/python27/lib/python2.7/subprocess.py", line 541, in check_call
2017-01-23 16:45:14,545 ERROR        raise CalledProcessError(retcode, cmd)
2017-01-23 16:45:14,546 ERROR    CalledProcessError: Command 'preprocess /tmp/tmpVFXClX.vcf.gz:* -l 4:46200230-49670140 -o /tmp/input.4:46200230-496701409qciL6.prep.vcf.gz -V 1 -L 0 -r /ssd/reference/fda/nih.GRCh37.fasta' returned non-zero exit status 1
2017-01-23 16:45:14,546 ERROR    ------------------------------------------------------------
2017-01-23 16:45:33,072 ERROR    One of the preprocess jobs failed
2017-01-23 16:45:33,072 ERROR    Traceback (most recent call last):
2017-01-23 16:45:33,072 ERROR      File "/opt/local/bin/hap.py", line 419, in <module>
2017-01-23 16:45:33,072 ERROR        main()
2017-01-23 16:45:33,073 ERROR      File "/opt/local/bin/hap.py", line 306, in main
2017-01-23 16:45:33,073 ERROR        args.gender)  # same gender as truth above
2017-01-23 16:45:33,073 ERROR      File "/opt/hap.py/bin/pre.py", line 202, in preprocess
2017-01-23 16:45:33,073 ERROR        haploid_x=gender == "male")
2017-01-23 16:45:33,073 ERROR      File "/opt/hap.py/lib/python27/Haplo/partialcredit.py", line 201, in partialCredit
2017-01-23 16:45:33,073 ERROR        raise Exception("One of the preprocess jobs failed")
2017-01-23 16:45:33,073 ERROR    Exception: One of the preprocess jobs failed```

**Comment by pkrusche** [comment]
It seems there is a problem with the query file, and that there is a place in the code I need to add output for an error message.

To get the message that is suppressed, could you run this command (the preprocess binary should be deployed in the same place as hap.py):

QUERY.vcf.gz should be bgzipped and indexed (or a bcf file with corresponding csi index, if not, preprocess will fail):

```bash
preprocess QUERY.vcf.gz:* -l 4:46200230-49670140 -o test.vcf.gz -V 1 -L 0 -r /ssd/reference/fda/nih.GRCh37.fasta
```
This command should fail as above, but also output a message telling us why. I'll add error output for this case to the to-do list in hap.py 0.3.7

**Comment by pkrusche** [comment]
This should be fixed in v0.3.7 now.
